Sources/App/Features/Past Sponsors/Models/Migrations/PastSponsorMigration.swift

```diff
@@ -0,0 +1,17 @@
+import Fluent
+
+class PastSponsorMigration: AsyncMigration {
```

Comment:
Would it be possible to update the seed.sql file with some examples of this model to make it easier to onboard?